### PR TITLE
ScoredVertex

### PR DESCRIPTION
**Pull request by SamanthaDawsonBanks**
Specialisation of the vertex that also carries scoring for AStar search

### CODE REVIEW

File: mcomp_Java/src/common/datatypes/map/griddedMap/ScoredVertex.java [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+    // TODO Auto-generated constructor stub
+  }
+
+  /**
+   * 
+   * @param current
+   * @param dest
+   * @return
+   */
+  public double calcF(Waypoint current, Waypoint dest) { // I'm not sure this should be in here but
```
Comment (HarryWJackson1996) [inline mcomp_Java/src/common/datatypes/map/griddedMap/ScoredVertex.java:41]: At the time it made sense to have a method to just do the calculation of Fx, however it was never used. Keep it for the time being and as I may use it when refactoring the code.

**Review by HarryWJackson1996 — APPROVED** [review]
Great work

**Review by stevempope — APPROVED** [review]
All good

**Comment by SamanthaDawsonBanks** [comment]
now with 100% less merge conflicts!

**Pull request merged**